### PR TITLE
Intermediate merge request before 0.3 #2

### PR DESCRIPTION
**Pull request by darkbibni**
## Major changes 

- Bounce mechanic (R (keyboard) or B (gamepad) )
- New sprites
- New animations
- One sided platform (press down to pass through)

See more here :
https://trello.com/c/rQ29SXcx/10-prog-part-v-03

Thanks for the review :) !

### CODE REVIEW

**Comment by darkbibni** [comment]
[PROG PART v 0.3](https://trello.com/c/rQ29SXcx/10-prog-part-v-03)

File: Assets/Scripts/Utility/KekeDreamlandTool.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+namespace KekeDreamLand {
+
+    public class KekeDreamlandTool : MonoBehaviour {
```
Comment (NoZip) [inline Assets/Scripts/Utility/KekeDreamlandTool.cs]: il y a un mix and match ici au niveau des accolades

File: Assets/Scripts/Gameplay/Mobs/AIBehaviour.cs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿using UnityEngine;
+
+public abstract class AIBehaviour : MonoBehaviour {
```
Comment (NoZip) [inline Assets/Scripts/Gameplay/Mobs/AIBehaviour.cs]: mix and match au niveau des accolades

File: Assets/Scripts/Utility/KekeDreamlandTool.cs (hunk 1 above)
Comment (darkbibni) [inline Assets/Scripts/Utility/KekeDreamlandTool.cs]: Qu'entends tu par mix and match ^^ ?

**Pull request merged**